vllm_ascend/model_runner.py

```diff
@@ -879,92 +879,6 @@ def set_in_profile_run(self):
         finally:
             self.in_profile_run = False
 
-    @torch.inference_mode()
-    def profile_run(self) -> None:
```

Comment:
please remove the one in NPUModelRunner, this base one will be used by other runner, such as pooling